### PR TITLE
Create michelonfray.com.xml

### PR DESCRIPTION
**Pull request by denisj**

### CODE REVIEW

File: src/chrome/content/rules/michelonfray.com.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="michelonfray.com">
+  <target host="michelonfray.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/michelonfray.com.xml]: + `www.michelonfray.com`

Comment (denisj) [inline src/chrome/content/rules/michelonfray.com.xml]: Should i put it even if the site redirect to michelonfray.com?

Comment (J0WI) [inline src/chrome/content/rules/michelonfray.com.xml]: Yes. We also want to secure this redirect.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**